itests/src/test/java/fi/metatavu/edelphi/test/ui/base/index/ProfileTestsBase.java

```diff
@@ -18,76 +15,37 @@ public void testLoginRequired() {
-    login(ADMIN_EMAIL);
-    navigate(PROFILE_PAGE);
-    assertSubscriptionLevelTexts("Educational", 1, 25); 
-  }
-  
-  @Test
   public void testSubscriptionLevelBasic() {
-    if (SUBSCRIPTION_LEVELS_DISABLED) {
-      return;  
-    }
-    
     updateUserSubscription(1l, "BASIC", toDate(2016, 1, 1), toDate(2016, 10, 10));
     login(ADMIN_EMAIL);
     navigate(PROFILE_PAGE);
-    assertSubscriptionLevelTexts("Basic", 1, 50);
+    assertSubscriptionLevelTexts("Basic");
   }
   
   @Test
-  public void testSubscriptionLevelPro() {
-    if (SUBSCRIPTION_LEVELS_DISABLED) {
-      return;  
-    }
-    
-    updateUserSubscription(1l, "PRO", toDate(2016, 1, 1), toDate(2016, 10, 10));
+  public void testSubscriptionLevelPlus() {
+    updateUserSubscription(1l, "PLUS", toDate(2016, 1, 1), toDate(2016, 10, 10));
     login(ADMIN_EMAIL);
     navigate(PROFILE_PAGE);
-    assertSubscriptionLevelTexts("Professional", 2, 100);
+    assertSubscriptionLevelTexts("Plus");
   }
-  
+
   @Test
-  public void testSubscriptionLevelUnlimited() {
-    if (SUBSCRIPTION_LEVELS_DISABLED) {
-      return;  
-    }
-    
-    updateUserSubscription(1l, "UNLIMITED", toDate(2016, 1, 1), toDate(2016, 10, 10));
+  public void testSubscriptionLevelPremium() {
+    updateUserSubscription(1l, "PREMIUM", toDate(2016, 1, 1), toDate(2016, 10, 10));
     login(ADMIN_EMAIL);
     navigate(PROFILE_PAGE);
-    waitAndAssertText(".profileSubscriptionLeveText", String.format("Your current subscription level is \"%s\".", "Unlimited"));
-    assertNotPresent(".profileSubscriptionFeaturesLabel", ".profileSubscriptionPanels", ".profileSubscriptionPanelists");
-    waitAndAssertText(".profileSubscriptionEnds", "your subscription ends Oct 10, 2016");
-    waitAndAssertText(".profileChangeSubscriptionText", "You may change or continue your subscription by clicking");
-    waitAndAssertText(".profileChangeSubscriptionLink", "here"); 
+    assertSubscriptionLevelTexts("Premium");
   }
-
-  private void assertSubscriptionLevelTexts(String level, int panels, int panelists) {
-    waitAndAssertText(".profileSubscriptionLeveText", String.format("Your current subscription level is \"%s\".", level));
-    waitAndAssertText(".profileSubscriptionFeaturesLabel", "Your subscription level allows you to:");
-    waitAndAssertText(".profileSubscriptionPanels", String.format("Be a manager in %d active panel (currently you are a manager in %d panels)", panels, 0));
-    waitAndAssertText(".profileSubscriptionPanelists", String.format("Have %d panelists in each panel", panelists));
+  
+  private void assertSubscriptionLevelTexts(String level) {
+    waitAndAssertText(".profileSubscriptionLevelText", String.format("Your current subscription level is %s.", level));
     waitAndAssertText(".profileSubscriptionEnds", "your subscription ends Oct 10, 2016");
+    /**
+    TODO: Re-enable after payment service is enabled in production
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)